src/routes/user/register/+page.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+	import type { IPagesState } from "./types";
+
+	// Forms
+	const form_first = import("./forms/1.svelte");
+	const form_second = import("./forms/2.svelte");
+	const form_third = import("./forms/3.svelte");
```

Comment:
This is a very stupid decision. Dont do this in a file-router based project

Reply:
Reverted on https://github.com/baseplate-admin/coreproject-v4-ui/pull/35/commits/13a44282ff3be3a022cf232d7e43f3ceda8d6a9b